CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+<img width="94" height="71" src="logo.svg" title="Autoprefixer logo by Anton Lovchikov">
+
+# Contributing to Autoprefixer
+Please feel free to contribute to Autoprefixer by submitting Pull Requests to add new features, ehancements, bug fixes, etc. If you're not sure whether a PR should be created for something, [open an issue](https://github.com/postcssAautoprefixer/issues) to ask about it first. Check the specific section depending on the type of contribution you want to make.
+
+ * [Getting started](#getting-started)
+ * [Adding a new prefix](#adding-a-prefix)
+ * [Changing an existing prefix](#changing-an-existing-prefix)
+ * [Fixing bugs](#fixing-bugs)
+ * [Filing issues](#filing-issues)
+ * [Running Tests](#running-tests)
+ * [Tips and guidelines](#guidelines-and-tips)
+
+Once your PR is accepted it will incorporated into the next main release of Autoprefixer.
+
+**Important**: Autoprefixer works by pulling data from [caniuse](https://caniuse.com/). Therefore it's important that support for a particular CSS feature is available there before we can enable support for it here. You can check the listing of features that need to be added [here](https://caniuse.com/issue-list).
+
+## Getting started
+Before you begin contributing make sure you have a [GitHub account](https://github.com/signup/free).
+* [Fork the repository](https://github.com/postcssAautoprefixer)
+* Clone a copy of it to your computer: `git clone https://github.com/USERNAME/autoprefixer`
+* run `yarn install` this will install all dependencies needed to run tests
```

Comment:
You say to use `yarn install` here but you don't explain how to install Yarn or Node.js in the first place.